src/core/styles/Colors.js

```diff
@@ -1,15 +1,19 @@
 const COLORS = {
+  ACCENT_COLOR: '#49a3f2',
```

Comment:
just a silly style thing, but it's generally good to be consistent across your hex colors with using either all uppercase letters or all lowercase letters, but not both :) in other words, i'd choose either lowercase or uppercase and write all colors here using that format for letters.